mart/attack/perturber/batch.py

```diff
@@ -13,7 +13,7 @@
 __all__ = ["BatchPerturber"]
 
 
-class BatchPerturber(Callback, torch.nn.Module):
+class BatchPerturber(torch.nn.Module):
```

Comment:
Since the `Callback` parent class is removed, the `on_run_start` method here is still needed?

Reply:
If this is not going to be a Callback, you shouldn't add the perturbation in the callbacks list in the Adversary's `IterativeGenerator`: https://github.com/IntelLabs/MART/blob/4bed8926fb91ee0b16d798936f0db8bfe8092190/mart/attack/adversary.py#L101-L103